ci/jenkins/test.sh

```diff
@@ -839,6 +852,23 @@ function run_conformance {
     kubectl rollout status daemonset/antrea-agent -n kube-system
 
     set +e
+    # Conformance DNS test cases fail in IPv6 Kind cluster because here CoreDNS pods are IPv6 pods 
```

Comment:
We usually use the word with upper case for K8s specific concepts, e.g.: Service, Pod, Deployment etc.
Please check this in your comments
```suggestion
    # Conformance DNS test cases fail in IPv6 Kind cluster because here CoreDNS Pods are IPv6 Pods 
```